ndk-svelte-components/src/lib/user/Nip05.svelte

```diff
@@ -37,16 +39,59 @@
         }
     }
 
-    const fetchProfilePromise = new Promise<NDKUserProfile>((resolve, reject) => {
+    async function validNip05(pubkey: string | undefined, nip05: string | undefined): Promise<boolean | undefined> {
```

Comment:
this needs be extracted out of the component, probably to `NDKUser`

user.validateNip05()